gdk/commands/test/RunCommand.py

```diff
@@ -57,12 +57,12 @@ def _should_download_nucleus_archive(self, _nucleus_path: Path) -> bool:
         """
         return _nucleus_path == Path(self._config.default_nucleus_archive_path).resolve() and not _nucleus_path.exists()
 
-    def _run_testing_jar(self) -> None:
+    def run_testing_jar(self) -> None:
         """
         Run the testing jar from the build folder using the configured options.
         """
-        # TODO: identify jar from the build output
-        _jar_path = str(self._test_build_directory().joinpath("uat-features-1.0.0.jar").resolve())
+        _jar_path = self._identify_testing_jar().__str__()
```

Comment:
```suggestion
        _jar_path = self._identify_testing_jar().relative_to(Path.cwd())
```
not sure how reliable string representation is

Reply:
```suggestion
        _jar_path = str(self._identify_testing_jar())
```
nit: